views/pages/claims/view.hbs

```diff
@@ -114,6 +115,15 @@
             </div>
         </div>
     {{/each}}
+   {{else}}
+      {{#if menuH.claimed}}
```

Comment:
isn't the if case reversed ?

Reply:
Could you please elaborate? I didn't quite get you. By the way, this was working as expected on my machine.

Reply:
The first if checks for a claim, and if it isn't there, the else gets hit which again has nested ifs checking for which tab is open (claimed, accepted, rejected) and accordingly gives a response.
Could you please elaborate what seems to be the problem? I didn't quite get you. By the way, this was working as expected on my machine.